assets/src/edit-story/app/layout/index.js

```diff
@@ -22,11 +23,11 @@ const Editor = styled.div`
 	bottom: 0;
 	min-height: calc(100vh - 32px);
 
-  display: grid;
-  grid:
-    "lib  head  head" ${ HEADER_HEIGHT }px
-    "lib  canv  insp" 1fr
-    / ${ LIBRARY_WIDTH }px 1fr ${ INSPECTOR_WIDTH }px;
+	display: grid;
+	grid:
+		"lib  head  insp" ${ HEADER_HEIGHT }px
+		"lib  canv  insp" 1fr
+		/ minmax(${ LIBRARY_MIN_WIDTH }px, ${ LIBRARY_MAX_WIDTH }px) 1fr minmax(${ INSPECTOR_MIN_WIDTH }px, ${ INSPECTOR_MAX_WIDTH }px);
```

Comment:
seems pretty good tbh: https://caniuse.com/#feat=mdn-css_properties_grid-template-rows_minmax

Not sure I understand what you're trying to do? A media query? If so, it would be adaptive but not responsive..

Reply:
Yep. Good enough.